### PR TITLE
Use go.opentelemetry.io vanity URL

### PR DESCRIPTION
**Pull request by dmathieu**
This switches the repository's URL from the GitHub one to the `go.opentelemetry.io/ebpf-profiler` vanity url.

Related: https://github.com/open-telemetry/opentelemetry-go-vanityurls/pull/23

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,19 @@ LDFLAGS := -X github.com//open-telemetry/opentelemetry-ebpf-profiler/vc.version=
...
+TOOLS = $(CURDIR)/.tools
+
+$(TOOLS):
+	@mkdir -p $@
+$(TOOLS)/%: $(TOOLS_MOD_DIR)/go.mod | $(TOOLS)
+	cd $(TOOLS_MOD_DIR) && \
+	go build -o $@ $(PACKAGE)
+
+PORTO = $(TOOLS)/porto
+$(TOOLS)/porto: PACKAGE=github.com/jcchavezs/porto/cmd/porto
```
[the remark this answers is not in the document]
Comment (florianl) [reply Makefile:60]: Why should we not use something like `go install github.com/jcchavezs/porto/cmd/porto@v0.6.0`?
I might miss something, but it is not obvious to me, which version is used with the current appraoch.

Personally I would prefer, if not yet another tool is required to build/lint/run this repository. Would there be a way to provide the same functionality without external tooling?

[the remark this answers is not in the document]
Comment (dmathieu) [reply Makefile:60]: The version is specified here: https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/164/files#diff-8770730bfa9304e1f62ea3ad904253c3653028e5fc741ab6b8221c2f03e69da3

There is no linter to ensure vanity imports are properly set (and it doesn't look like porto can be used with golangci-lint).
This approach is the one used by the Go SDK: https://github.com/open-telemetry/opentelemetry-go/blob/main/Makefile#L221
As well as the collector: https://github.com/open-telemetry/opentelemetry-collector/blob/main/Makefile#L64

[the remark this answers is not in the document]
Comment (dmathieu) [reply Makefile]: I have moved porto back into a manual `go install` in the Makefile. We can revisit this later if we want proper upgrades I guess.

**Review by athre0z — APPROVED** [review]

**Review by florianl — APPROVED** [review]
I'm aware of the git conflict in `main.go`, but I don't see why resolving this conflict would change this proposal. So :+1: 

Thanks for the work! :pray:

**Review by fabled — APPROVED** [review]
LGTM. Though seems there's a merge conflict now. Could you rebase this on top of current main? Thanks!

**Review by rockdaboot — APPROVED** [review]
LGTM

**Pull request merged**